app/cmd/start.go

```diff
@@ -183,6 +208,10 @@ func start(c *cli.Context) (err error) {
 		if err := cleanupStaledNvmeAndDmDevices(); err != nil {
 			return err
 		}
+	} else {
+		if err := unfreezeFilesystems(); err != nil {
+			return err
+		}
```

Comment:
Can we do this during IM startup? As we discussed before, after the upgrade, will the new IM pod wrongly unfreeze fs handled by old IM pods?

Reply:
It will. However, I think the benefits outweigh the risks:

- Since snapshots are very fast, there will likely not be any ongoing in the old instance manager during the restart.
- If there ARE freezing file systems in another instance manager, we cannot stop their freeze, but we might unfreeze them immediately after it completes. This means their data will be flushed, but the snapshot MIGHT be taken after some I/O manages to resume. This is exactly the same behavior we have today, so the situation only improves with the feature addition.
- The worst thing we can do with this feature (IMO) is leave froze file systems stranded on nodes. It will lead to unhappy users and extra support engagements.

Reply:
OK. Got it! I agree that the benefits outweigh the risks, and things won't be worse than the today's implementation